CHANGELOG.md

```diff
@@ -96,7 +96,7 @@
 * Realm Studio: 15.0.0 or later.
 
 ### Internal
-* Using Core 14.1.0.
+* Using Core 14.4.1.
```

Comment:
Forgot this before, but we should also update the changelog with relevant core changes.

Reply:
Ok. Think it makes sense for @papafe  to do it, since you can better judge what to include.

Reply:
Nikola just updated the changelog and core in another PR, so no need to do it